src/lib/components/_utilities/locale-strings.svelte

```diff
@@ -1,5 +1,5 @@
 <script lang="ts">
-  import { interpolate } from '$lib/utilities/i18n.utilities';
+  import { interpolate } from '$lib/_utilities/i18n.utilities';
```

Comment:
You're going to see quite a bit of these path and directory changes. I wanted to create a consistent and obvious way to declare utilities for a group of components. This helps bring that utilities directory to the top of the file list, and makes it obvious that it stands apart from the regular components.